petab/lint.py

```diff
@@ -108,6 +108,10 @@ def check_condition_df(
 
     check_ids(df.index.values, kind='condition')
 
+    if not df.index.is_unique:
```

Comment:
Maybe this is inappropriate according to the docs, and could be replaced with `if df.index.has_duplicates`
```python
>>> print(pd.Index.is_unique.__doc__)

        Return if the index has unique values.
        
>>> print(pd.Index.has_duplicates.__doc__)

        Check if the Index has duplicate values.
```

Reply:
The tests may have passed because there are always only duplicated or only unique values in any tested index (no mixture).

Reply:
I think the docstring there maybe slightly misleading. `has_duplicates` is only `return not self.is_unique `. So, I'd directly use the latter.